frontend/package.json

```diff
@@ -87,6 +87,7 @@
     "eslint-config-prettier": "6.7.0",
     "eslint-plugin-prettier": "3.1.1",
     "husky": "4.2.3",
+    "jest-sonar-reporter": "2.0.0",
```

Comment:
We need `jest-sonar-reporter` to pass unit test results to SonarQube; e.g "100 test executed, 80 pass, 20 skipped, etc"

This is different than code coverage, which we already were passing down

Reply:
I'm assuming this is a dev dependency?

Reply:
Yes it is